src/shipit_static_analysis/default.nix

```diff
@@ -193,7 +193,7 @@ let
       update = writeScript "update-${name}" ''
         pushd ${self.src_path}
         ${pypi2nix}/bin/pypi2nix -v \
-          -V 3.5 \
+          -V 3.6 \
```

Comment:
You also need to update the lines above in Mozlint deps

Reply:
Done!